about.html

```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="badakhsh portfolio, portfolio" />
+
+    <!-- Links -->
+    <link href="style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link
+      rel="project icon"
+      href="./assets/logos/Logo.png"
+      type="image/x-icon"
+    />
+
+    <!-- Project Title -->
+    <title>Capstone Project</title>
+  </head>
+  <body>
+    <header>
+      <!--Header section-->
+      <div class="header">
+        <div class="quicklink social">
+          <a href="#"
+            ><img src="assets/logos/linkedinicon.png" alt="linked In Icon"
+          /></a>
+        </div>
+        <div class="quicklink social">
+          <a href="#"
+            ><img src="assets/logos/Twitter.png" alt="twitter logo"
+          /></a>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">English</a></p>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">My Page</a></p>
+        </div>
+        <div class="quicklink">
+          <p><a href="#">Logout</a></p>
+        </div>
+      </div>
+
+      <!-- Navbar Section -->
+      <div class="humberger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+      </div>
+      <div id="navbar" class="navbar">
+        <div class="logo">
+          <a href="index.html"
+            ><img src="assets/logos/Logo.png" alt="Page logo"
+          /></a>
+        </div>
+        <ul>
+          <li class="links home-page"><a href="index.html">Home</a></li>
```

Comment:
here is a link for the Home Page while switching to mobile and it's display set to none in desktop.